gamemodes/horde/gamemode/sv_commands.lua

```diff
@@ -99,6 +99,18 @@ function End(ply)
     ply:PrintMessage(HUD_PRINTTALK, "Stopping game...")
 end
 
+function NearBeacon(ply)
+    local beacons = ents.FindByClass("horde_watchtower_beacon")
+    if beacons and #beacons > 0 then
```

Comment:
ents.FindByClass always returns a table, it'd be better to early return here. You'd have to fix the `end` and scoping once you accept this
```suggestion
    if #beacons == 0 then return false end
```

Reply:
added this

Reply:
you still check if beacons exists